backend/src/index.js

```diff
@@ -25,46 +24,11 @@ app.use("/squads", routes.squad);
 sequelize.sync({ force: eraseDatabaseOnStart }).then(() => {
   if (eraseDatabaseOnStart) {
     console.log("Database erased. Repopulating.");
-    createSampleUsers();
+    populateDatabase();
   }
   app.listen(process.env.PORT, () => {
     console.log(`frontend is listening on port ${process.env.PORT}`);
   });
 });
 
-// Populate database functions
```

Comment:
Moved this to its own folder. Will be turning this into a script to setup a good sample database both for testing and for development.